indico/modules/events/layout/templates/layout_conference.html

```diff
@@ -28,6 +28,11 @@
                 </a>
             {% endcall %}
         {% endcall %}
+        {% if theme_settings_form %}
```

Comment:
Maybe this should be in a dialog (and only available once the theme has been saved for the event, since otherwise it's confusing if you change the theme but still see another theme's config form)?